app/src/modules/analytics/events/common/rules/index.js

```diff
@@ -118,6 +118,11 @@ export const trackErrorInRuleCreation = (description, rule_type) => {
   trackEvent(RULES.ERROR_IN_RULE_CREATION, params);
 };
 
+export const trackRuleEditorViewed = (source, ruleType) => {
+  const params = { source, ruleType };
```

Comment:
`ruleType` should be `snake_case`
```suggestion
export const trackRuleEditorViewed = (source, rule_type) => {
  const params = { source, rule_type };
```